### PR TITLE
743 january acep updates

### PR DESCRIPTION
**Pull request by vpaneri**
Categorized treatment into Admission, Discharge instead of individual treatment type. Steriods are still separate since it is part of the 'Do not use'  treatment that is displayed differently on the UI.

The named expression returns a boolean or null instead of strings (`use` `do-not-use`), which makes the logic no the UI simplier. 

Ticket: [743](https://app.zenhub.com/workspaces/covid-patient-manager-5fc9453d958e1600153602d5/issues/department-of-veterans-affairs/covid-patient-manager/743)

### CODE REVIEW

File: cql-unit-tests/test/treatmentSummary.test.ts [hunk 1]
```diff
@@ -11,11 +11,9 @@ import { buildAllNullRiskAssessmentScoreParameters } from '../helpers/builders';
 describe('treatment summary', () => {
     it.each([
         ['Recommend Non-Pharmacologic Treatment'],
-        ['Recommend Antibodies Treatment'],
-        ['Recommend Anticoagulation Treatment'],
-        ['Recommend Steroids Treatment'],
-        ['Recommend Remdesivir Treatment'],
-        ['Recommend SteroidsAndOrRemdesivir Treatment'],
+        ['Recommend Admission Treatment'],
+        ['Recommend Discharged Treatment'],
```
Comment (vpaneri) [inline cql-unit-tests/test/treatmentSummary.test.ts:15]: Discussed with @rocio-desantiago in the zoom to rename this to `Recommend Discharge Treatment` for clarity.

File: cql-unit-tests/test/treatmentSummary.test.ts [hunk 2]
```diff
@@ -39,10 +37,10 @@ describe('treatment summary', () => {
     });
 
     it.each([
-          ['mild', 'use', new ClinicalAssessmentBuilder().withMildSeverity().build()],
-          ['moderate', 'use', new ClinicalAssessmentBuilder().withConcerningLab(1).withMildSeverity().build()],
-          ['severe', 'use', new ClinicalAssessmentBuilder().withSevereSeverity().build()],
-          ['critical', 'use', new ClinicalAssessmentBuilder().withCriticalSeverity().build()],
+          ['mild', true, new ClinicalAssessmentBuilder().withMildSeverity().build()],
```
Comment (ddurhamflexion) [inline cql-unit-tests/test/treatmentSummary.test.ts:40]: 👍 `true`

File: cql-unit-tests/test/treatmentSummary.test.ts [hunk 3]
```diff
@@ -52,34 +50,34 @@ describe('treatment summary', () => {
...
-    )('For %p severity, Recommend Antibodies Treatment is %p', (severityType: string, expectedRecommendation: string, clinicalAssessmentOverrides: Partial<ClinicalAssessmentsParameters>) => {
+    )('For %p severity, Recommend DischargeHome Treatment is %p', (severityType: string, expectedRecommendation: string, clinicalAssessmentOverrides: Partial<ClinicalAssessmentsParameters>) => {
         const cqlExpressionParameters = buildCQLExpressionParameters(clinicalAssessmentOverrides);
-        const recommendNonPharma = executeSummaryCQLExpression(cqlExpressionParameters, 'Recommend Antibodies Treatment');
+        const recommendNonPharma = executeSummaryCQLExpression(cqlExpressionParameters, 'Recommend Discharge Treatment');
         expect(recommendNonPharma).toEqual(expectedRecommendation);
     });
 
     it.each([
             ['mild', null, new ClinicalAssessmentBuilder().withMildSeverity().build()],
```
Comment (ddurhamflexion) [inline cql-unit-tests/test/treatmentSummary.test.ts:66]: i defer. and wondering we prefer `false` to `null`. No problem either way to me

File: cql-unit-tests/test/treatmentSummary.test.ts [hunk 4]
```diff
@@ -52,34 +50,34 @@ describe('treatment summary', () => {
...
         const cqlExpressionParameters = buildCQLExpressionParameters(clinicalAssessmentOverrides);
-        const recommendNonPharma = executeSummaryCQLExpression(cqlExpressionParameters, 'Recommend Anticoagulation Treatment');
+        const recommendNonPharma = executeSummaryCQLExpression(cqlExpressionParameters, 'Recommend Admission Treatment');
         expect(recommendNonPharma).toEqual(expectedRecommendation);
     });
 
     it.each([
-            ['mild', 'do-not-use', new ClinicalAssessmentBuilder().withMildSeverity().build()],
-            ['moderate', 'do-not-use', new ClinicalAssessmentBuilder().withModerateSeverity().withConcerningLab(1).build()],
+            ['mild', false, new ClinicalAssessmentBuilder().withMildSeverity().build()],
```
Comment (ddurhamflexion) [inline cql-unit-tests/test/treatmentSummary.test.ts:79]: 👍 on using `false`

File: cql-unit-tests/test/treatmentSummary.test.ts (hunk 3 above)
Comment (vpaneri) [reply cql-unit-tests/test/treatmentSummary.test.ts:66]: I agree it's confusing, but currently, treatments are recommended (true), not recommended(false), and not applicable(null). I also deferred the change when we have a better grasp of how `HasAdmissionOrDischargeRecommendation` can be used.

**Pull request merged**